packages/core-mobile/app/store/network/listeners.ts

```diff
@@ -9,17 +9,25 @@ import {
 } from 'store/settings/advanced'
 import { AnyAction, isAnyOf } from '@reduxjs/toolkit'
 import AnalyticsService from 'services/analytics/AnalyticsService'
-import { selectCustomNetworks, selectFavorites } from './slice'
+import { isDevnet } from 'utils/isDevnet'
+import {
+  selectActiveNetwork,
+  selectCustomNetworks,
+  selectFavorites
+} from './slice'
 
 const adjustActiveNetwork = (
   _: AnyAction,
   listenerApi: AppListenerEffectAPI
 ): void => {
   const { dispatch, getState } = listenerApi
   const state = getState()
+  const activeNetwork = selectActiveNetwork(state)
   const isDeveloperMode = selectIsDeveloperMode(state)
 
-  const chainId = isDeveloperMode
+  const chainId = isDevnet(activeNetwork)
+    ? ChainId.AVALANCHE_DEVNET_ID
```

Comment:
I don't think we want this logic cause it won't switch between mainnet <-> testnet

Reply:
removed.